src/romi-robot.ts

```diff
@@ -534,7 +577,11 @@ export default class WPILibWSRomiRobot extends WPILibWSRobotBase {
                 encoderValue = buf.readInt16LE();
 
                 const lastValue = encoderInfo.lastRobotValue;
-                const delta = encoderValue - lastValue;
+
+                // Figure out if we should be reporting flipped values
+                const reverseMultiplier = (encoderInfo.isHardwareReversed ? -1 : 1) *
```

Comment:
I think you are assuming isHardwareReversed and isSoftwareReversed as created fields. I think this should have an:
If encoderInfo.isHardwareReversed != undefined && encoderinfo.isSoftwareReversed != underfined ....

Reply:
It's actually fine, since `undefined` is false-y, so not having the field present is the same as being `false`